src/bluetooth/TemperatureLogManager.js

```diff
@@ -65,7 +66,7 @@ export class TemperatureLogManager {
 
     return logsToSave.map(({ temperature }, i) => {
       const offset = logInterval * i;
-      const timestamp = Number(moment(initial).add(offset, 's').format('X'));
+      const timestamp = Number(moment(initial).add(offset, 's'));
```

Comment:
Logs were being generated in the 1970s (pretty sure the sensor wasn't around then!).
Alternatively could multiply the offset by 1000 and then `format('X')`?

Reply:
Hm yeah, CCA using just seconds. Good to use ms though